_posts/2024-02-01-geospatial-techno.md

```diff
@@ -0,0 +1,107 @@
+- Check the "Enabled" checkbox to have separate metadata at the workspace level and global service for this workspace.
+- Use the checkbox located next to each service to override the Global Settings for the associated service. Once enabled, clicking on the service link will open the settings page for the service, allowing default values for the service title, abstract, and other details to be supplied.
+- The Security tab allows to set data access rules at the workspace level. To create/edit the workspace's data access rules, check/uncheck checkboxes according to the desired role.
+
+#### Remove a workspace
+To remove a workspace, select it by clicking the checkbox next to the workspace. Multiple workspaces can be selected, or all can be selected by clicking the checkbox in the header. Click the "Remove selected workspace(s)" button. You will be asked to confirm or cancel the removal. Clicking OK removes the selected workspace(s).
+
+## Stores
+GeoServer's Stores serve as intermediaries between the server and the data sources where your spatial data resides. They provide a mechanism for GeoServer to connect to various data repositories, including file systems, databases, and cloud storage services. Each store represents a unique data source and has its configuration settings.
+
+#### Add a store
+To add a Store, click "Stores" from the "Data" section. Click the "Add new store" button. You will be prompted to choose a data source. GeoServer supports several different data formats, but they are classified into three types: Vector data, Raster data, and Cascaded Services.
+- Vector data formats include Shapefile, GeoPackage, PostGIS, and Properties. The most common and widely used option is Shapefile.
+- Raster data formats include ArcGrid, GeoPackage, GeoTIFF, ImageMosaic, and WorldImage. The most used and well-known are the GeoTIFF and the WorldImage. GeoTIFF is a spatial extension of the "TIFF" format and tags image files with geographic information. A WorldImage is similar, but georeferencing information is saved in an external text file.
+- Cascaded Services include WFS, WMS, and WMTS. GeoServer can proxy a remote Web Map Service and Web Map Tile Service and load it as a store in GeoServer.
+
+Other data sources are supplied as GeoServer extensions. Extensions are downloadable modules that add functionality to GeoServer. Click the appropriate data source to configure the store, because the connection parameters vary depending on data format.
+
+To create a Shapefile data store, follow these steps:
+- Select the desired workspace from the drop-down menu.
+- Enter the Data Source Name. Make sure the Enabled option is checked. Otherwise, access to the store along with all the datasets defined, will be disabled for it.
+- In the "Shapefile location", click on the "Browse" link to define the location of the shapefile.
+- DBF charset is used to decode strings from the dbf file, so it's very important to select an appropriate DBF character set.
```

Comment:
I rarely have to use this option; I think it is only needed as an "override" when the shape file could not figure it out?

> DBF charset can be supplied, the character set could not be determined correctly from the DBF file